ai_genomics/analysis/gtr_definitions.py

```diff
@@ -56,13 +54,14 @@
     ai_projs_abstr, genom_projs_abstr = [
         set(
             [
-                element["id"]
-                for element in projects
+                gtr_id
+                for gtr_id, abstract_text, title in zip(
+                    projects["id"].values(),
+                    projects["abstractText"].values(),
+                    projects["title"].values(),
+                )
                 if any(  # If any of the abstract terms are in the abstract or title
-                    t
-                    in str(element["abstractText"]).lower()
-                    + str(element["title"]).lower()
-                    for t in terms
+                    t in str(abstract_text).lower() + str(title).lower() for t in terms
```

Comment:
```suggestion
                    t in abstract_text.lower() + title.lower() for t in terms
```

Do any risks arise from searching for the terms inside a string rather than searching a list of terms in the string (e.g. matching to partial words)?

Reply:
These need to have `str()` as there some records read in as ints.

Good point about searching within a string vs. searching terms. I'll create an issue relating to it 👍